view/theme/frio/css/image_grid.css

```diff
@@ -0,0 +1,43 @@
+* {
+	box-sizing: border-box;
+}
+
+.row {
+	display: -ms-flexbox; /* IE10 */
+	display: flex;
+	-ms-flex-wrap: wrap; /* IE10 */
+	flex-wrap: wrap;
+	padding: 0 4px;
+}
+
+/* Create four equal columns that sits next to each other */
+.column {
+	-ms-flex: 50%; /* IE10 */
+	flex: 50%;
+	max-width: 50%;
+	padding: 0 4px;
+}
+
+.column img {
+	margin-top: 8px;
+	vertical-align: middle;
+	width: 100%;
+}
+
+/*!* Responsive layout - makes a two column-layout instead of four columns *!*/
+/*@media screen and (max-width: 50px) {*/
+/*	.column {*/
+/*		-ms-flex: 50%;*/
+/*		flex: 50%;*/
+/*		max-width: 50%;*/
+/*	}*/
+/*}*/
+
+/* Responsive layout - makes the two columns stack on top of each other instead of next to each other */
+@media screen and (max-width: 150px) {
```

Comment:
What's the case where a screen would be narrower than 150px?

Reply:
I thought it might be a good idea to a have only one column then. But I have to admit I never actually tested it, and now I see that values < 150px are never reached :-)

Reply:
I decided that it is not needed and removed the "switch back to one column" in general.